skrf/media/cpw.py

```diff
@@ -25,50 +24,198 @@
 
 class CPW(Media):
     """
-    A Coplanar Waveguide.
-
-    This class was made from the technical documentation [#]_ provided
-    by the qucs project [#]_ .
-    The variables  and properties of this class are coincident with
-    their derivations.
+    A coplanar waveguide transmission line is defined in terms of width,
+    spacing, and thickness on a given relative permittivity substrate of a
+    certain height. The line has a conductor resistivity and a tangential loss
+    factor. The backside of the strip can be made of air or metal (grounded 
+    coplanar waveguide).
+
+    This class is highly inspired by the technical documentation [QUCSa]_
+    and sources provided by the qucs project [QUCSb]_ .
+    
+    In addition, Djordjevic [DBLS01]_ /Svensson [SvDe01]_  wideband debye dielectric
+    model is considered to provide a more realistic modelling of broadband
+    microstrip with causal time domain response.
+    
+    A compatibility mode is provided to mimic the behaviour of QUCS or of
+    Keysight ADS. There is known differences in the output of these
+    simulators.
+    
+    The quasi-static models of characteristic impedance and effective
+    permittivity give the value at zero frequency. The dispersion models
+    compute frequency-dependant values of these variables.
+    
+    * Quasi-static characteristic impedance and effective permittivity model
+      use [GhNa84]_ and [GhNa83]_. The models are corrected to account for
+      strip thickness using a first-order approach described in [GGBB96].
+      A comparison shows that ADS simulator uses another thickness correction
+      method that is according to ADS doc based on [Cohn60]. This second method
```

Comment:
Added underscores for sphinx references:

```suggestion
      strip thickness using a first-order approach described in [GGBB96]_.
      A comparison shows that ADS simulator uses another thickness correction
      method that is according to ADS doc based on [Cohn60]_. This second method
```